<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>109</width>
    <height>299</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>119</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QWidget" name="scl_min" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: black;
border-width: 1px;
border-bottom-style: none;
border-left-style: none;
border-right-style: none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="Scale_rng_chk_box" name="min_chk">
        <property name="styleSheet">
         <string notr="true">border-style: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Scale_rng_input" name="min_val">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string>minimum</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{
	border: 1px solid black;
}
QLineEdit:!enabled{
	border: 1px solid black;
	background-color: rgba(0, 0, 0, 15);
	color: black;
}</string>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="scl_max" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>75</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-style: solid;
border-color: black;
border-width: 1px;
border-top-style: none;
border-left-style: none;
border-right-style: none;</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="Scale_rng_chk_box" name="max_chk">
        <property name="styleSheet">
         <string notr="true">border-style: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Scale_rng_input" name="max_val">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit:enabled{
	border: 1px solid black;
}
QLineEdit:!enabled{
	border: 1px solid black;
	background-color: rgba(0, 0, 0, 15);
	color: black;
}</string>
        </property>
        <property name="text">
         <string>---</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="gradient_spacer_v4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <spacer name="gradient_spacer_v3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="3">
    <spacer name="gradient_spacer_v2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="gradient_spacer_v1">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="grad_btn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Scale_rng_input</class>
   <extends>QLineEdit</extends>
   <header>Viewer_Widgets.h</header>
  </customwidget>
  <customwidget>
   <class>Scale_rng_chk_box</class>
   <extends>QCheckBox</extends>
   <header>Viewer_Widgets.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>max_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>max_val</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>min_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>min_val</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grad_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>scl_max</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>24</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>grad_btn</sender>
   <signal>toggled(bool)</signal>
   <receiver>scl_min</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
